vm/x86/x86emu/src/emulator.rs

```diff
@@ -675,10 +710,10 @@ impl<'a, T: Cpu> Emulator<'a, T> {
             }
         }
         tracing::trace!(
-            bytes = ?self.bytes[..instr.len()],
-            cs = ?self.state.segs[CpuState::CS],
-            rip = self.state.rip,
-            bitness = ?self.state.bitness(),
+            code = ?instr.code(),
```

Comment:
We don't trace the code here on purpose for binary size. Iced's debug impl for the code enum is extremely large.